src/features/rewards/mobile/selectMobile/style.ts

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components'
+
+interface StyleProps {
+  floating?: boolean
+}
+
+export const StyledSelect = styled<StyleProps, 'select'>('select')`
+  width: 100%;
+  background: #fff;
+  height: 34px;
+  font-size: 14px;
+  border: ${p => p.floating ? 'none' : '1px solid #DFDFE8'};
+  text-align-last: ${p => p.floating ? 'right' : 'left'};
+
+  &:focus {
+    outline: 0;
+  }
+`
```

Comment:
@rossmoody I experimented by adding `dir={'rtl'}` to `StyledSelect` (which computes in CSS to `direction: rtl;`) and this was my result:

<img width="466" alt="screen shot 2019-03-06 at 11 47 48 pm" src="https://user-images.githubusercontent.com/8732757/53937473-767b1680-406a-11e9-9c48-06aa1db945d7.png">

I think the intended behavior for `direction` at that setting is a bit different than what we are looking for. Please let me know if you implemented this a different way and got the desired results, or if I otherwise misunderstood your feedback, thanks!

Reply:
Yeah, I was wrong (way wrong). It doesn't seem great tying removing the border and justifying right to the same prop but I see it's creating a specific look. Might also want to consider removing the width=100% in that instance so it renders more like a button inline. Either way, I'm good.